src/compiler/program.ts

```diff
@@ -1290,10 +1290,21 @@ namespace ts {
             });
         }
 
+        function shouldTreatWarningsAsErrors() {
+            const setValue = program.getCompilerOptions().treatWarningsAsErrors;
+            if (typeof setValue === "undefined") {
+                return true;

```

Comment:
Nice that it defaults to the safer setting (`true`); but how do you _disable_ it via command-line arguments? `--no-treatWarningsAsErrors`?

Reply:
`--treatWarningsAsErrors false`, same as our other boolean flags.